web/src/pages/Route/components/DebugViews/DebugDrawView.tsx

```diff
@@ -119,7 +119,8 @@ const DebugDrawView: React.FC<RouteModule.DebugDrawProps> = (props) => {
   };
 
   const handleDebug = (url: string) => {
-    if (url === '') {
+    const urlReg = /^(?=^.{3,255}$)(www\.)?[a-zA-Z0-9][-a-zA-Z0-9]{0,62}(\.[a-zA-Z0-9][-a-zA-Z0-9]{0,62})+(:\d+)*(\/\w+\.\w+)*([\?&]\w+=\w*)*$/;
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/103351148-39083880-4add-11eb-8d20-7fbb19e7ff8d.png)

Wow

Reply:
> ![image](https://user-images.githubusercontent.com/2106987/103351148-39083880-4add-11eb-8d20-7fbb19e7ff8d.png)
> 
> Wow

love this picture~! current reg has some errors, updating